src/main/java/uk/ac/ebi/ampt2d/metadata/rest/controllers/StudyRestController.java

```diff
@@ -108,9 +152,38 @@
         return ResponseEntity.ok(resources);
     }
 
+    @ApiOperation(value = "Update an existing study")
+    @ApiParam(name = "id", value = "Study's id (accession.version)", type = "string", required = true, example = "EGAS0001.1")
+    @RequestMapping(method = RequestMethod.PATCH, path = "{id}/patch", produces = "application/json", consumes = "application/json")
+    @ResponseBody
+    @SuppressWarnings("unchecked")
+    public ResponseEntity<Resource<Study>> patch(@PathVariable("id") AccessionVersionEntityId id,
+                                                 @RequestBody String json) {
```

Comment:
Can we document in swagger , the difference between normal patch and this custom patch.

Reply:
@RequestBody Study study instead of String json ?

Reply:
I don't know if the documentation was changed after this comment, but I don't understand what "especially for those deprecated or unpublished studies" means. It still doesn't explain what is different from the regular patch.

Reply:
Added more details in 1db44da